prep-css-display/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="styles.css">
+  <title>CSS Display Properties</title>
+  <style>
+
+  </style>
+</head>
+
+<body>
+
+  <div class="blue-border">I am a div and by default I am block, which means I take up all the space I can horizontally.</div>
+
+  <div>
+    I am a div but I have <span style="border: 4px solid green;">span elements inside me</span>.
+  Span elements are <span style="border: 4px solid green;">display inline by default</span>,
+  so they only take up as much as space as they need.
```

Comment:
Hi Charlene, for this one we want to use CSS `span` selectors to style these elements instead of the `style` attribute.